### ISSUE TITLE
Fetch failed with 404 - "gulp-accessibility"

### ISSUE THREAD

**Issue by KrisOlszewski** [npm/npm#8440]
Can't install `gulp-accessibility` via CLI. See below.

**Error output**

```
npm i gulp-accessibility --save-dev
npm ERR! fetch failed https://registry.npmjs.org/gulp-accessibility/-/gulp-accessibility-1.1.1.tgz
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 404
npm ERR! fetch failed https://registry.npmjs.org/gulp-accessibility/-/gulp-accessibility-1.1.1.tgz
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 404
npm ERR! fetch failed https://registry.npmjs.org/gulp-accessibility/-/gulp-accessibility-1.1.1.tgz
npm ERR! Darwin 13.4.0
npm ERR! argv "node" "/usr/local/bin/npm" "i" "gulp-accessibility" "--save-dev"
npm ERR! node v0.12.4
npm ERR! npm  v2.10.1

npm ERR! fetch failed with status code 404
```

**Comment by KrisOlszewski** [comment]
Fixed by module author (https://github.com/yargalot/gulp-accessibility/issues/2).
